src/utils/updateWiki.ts

```diff
@@ -0,0 +1,253 @@
+import { statSync } from 'fs';
+import { appendFile, writeFile, readdir, readFile } from 'fs/promises';
+
+// File Paths
+const wikiPath = 'docs/COMMAND-WIKI.md';
+const commandsDir = 'src/commands';
+const commandDetailsDir = 'src/commandDetails';
+
+// Commands with subcommands, to be handled separately
+const compoundCommands = ['coin', 'company', 'interviewer', 'leetcode', 'profile'];
```

Comment:
Can you find a way to automate this? I know its too much to ask, but it would really help, since everytime we create a new command we wouldn't need to update here. Thanks